controllers/mapController.js

```diff
@@ -46,5 +46,36 @@ const createNewMap = async (req, res, next) => {
   }
 };
 
+const getMembers = async (req, res, next) => {
+  const { id } = req.params;
+
+  try {
+    const { members: memberList } = await Map.findById(id)
+      .populate("members")
+      .lean()
+      .exec();
+    const members = [];
+
+    memberList.map(member => {
+      return members.push({
+        profileUrl: member.profileUrl,
+        nickname: member.nickname,
+      });
+    });
```

Comment:
map 메소드는 배열의 요소를 순회하며 반환한 값들을 요소로 가지는 새로운 배열을 반환하는 값을 따로 변수로 받고 있지 않고 있어서 어색해보이는 것 같아요! push만 하는 로직으로 한다면 forEach 메소드로 변경해주는게 좋을 것 같은데 다른 분들의 의견도 궁금합니다! 
```suggestion
    memberList.forEach(member => {
      members.push({
        profileUrl: member.profileUrl,
        nickname: member.nickname,
      });
    });
```


map 메소드를 사용하려면 아래와 같이 members에 할당해주도록 수정이 필요할 것 같습니다! 저는 개인적으로 map 메소드를 쓰는 쪽으로 수정하면 좋을 것 같다는 생각입니다~!
```suggestion
    const members = memberList.map(member => {
      return {
        profileUrl: member.profileUrl,
        nickname: member.nickname,
      });
    });
```